vms/proposervm/batched_vm_test.go

```diff
@@ -54,7 +57,11 @@ func TestCoreVMNotRemote(t *testing.T) {
 
 func TestGetAncestorsPreForkOnly(t *testing.T) {
 	require := require.New(t)
-	coreVM, proRemoteVM, coreGenBlk := initTestRemoteProposerVM(t, mockable.MaxTime) // disable ProBlks
+	var (
+		activationTime = mockable.MaxTime
+		durangoTime    = time.Unix(0, 0)
```

Comment:
@StephenButtolph a vm can activate Snowman++ after durango has activated. As such I think it makes sense that activation time follows Durango.
In general I want to keep as many tests as possible update to the latest fork. I'd do this in this PR,  but it could be made a post activation cleanup too.